pkg/util/utils_supported.go

```diff
@@ -60,6 +60,36 @@ func GetRootlessRuntimeDir() (string, error) {
 	return rootlessRuntimeDir, nil
 }
 
+// GetRootlessConfigHomeDir returns the config home directory when running as non root
+func GetRootlessConfigHomeDir() (string, error) {
+	var rootlessConfigHomeDirError error
+
+	rootlessConfigHomeDirOnce.Do(func() {
+		home := os.Getenv("HOME")
+		resolvedHome, err := filepath.EvalSymlinks(home)
```

Comment:
Thought: is this necessary? it usually leads to inconsistent paths if `$HOME` happens to be a symlink. Also sometimes `$HOME` is defined but not created, with the expectation that the app will `mkdir -p $HOME/.../app-dir` anyway.

Reply:
Leaving as is for now, @giuseppe thoughts?